AMP-70464 update cohort API with latest info + new sync metadata parameter (#680)
Amplitude-Developer-Docs/amplitude-dev-center: docs/analytics/apis/behavioral-cohorts-api.md

    Authorization: Basic {{api-key}}:{{secret-key}} #credentials must be base64 encoded
    ```

### Get all cohorts query parameters

| Name|Description|
|----|-----|
|`includeSyncInfo`|<span class="optional">Optional</span>. Boolean. Set to true to include cohort sync metadata in response (one-time + disabled sync will be excluded) .|

> !!!note "Notes about query parameters"
> 
>     - This feature is currently in Beta and require whitelist, please contact Amplitude Support, or your Amplitude account manager if you need this

Reviewer: ```suggestion
!!!beta "Notes about query parameters"

    - This feature is currently in Beta and requires whitelisting. Please contact Amplitude Support, or your Amplitude account manager for access.
```
